src/clj/witan/send/charts.clj

```diff
@@ -107,4 +107,74 @@
                                                                    :values (r-combine colours)}]]
                                              [[:scale_x_continuous {:breaks (r-combine (filter even? (range min-year max-year)))
                                                                     :limit [:c min-year max-year]}]]))]))
-  (move-file "Rplots.pdf" (str "target/" title "_probability.pdf")))
+  (move-file "Rplots.pdf" (str "target/" title "_Probability.pdf")))
+
+(defn get-total-historical-send-pop [year transitions-data]
+  (count (map :academic-year-1 (filter #(= (:calendar-year %) year) (filter #(not= (:setting-1 %) :NONSEND) transitions-data)))))
+
+(defn is-send-and-is-year? [row year]
+  (and (not= (:setting-1 row) :NONSEND)
+        (= (:calendar-year row) year)))
+
+(defn foo [year transitions-data]
```

Comment:
Perhaps a better name? :)

Reply:
Ooops That's one demo function you wrote. I need to take that out.